doc/nrf/app_dev/device_guides/nrf91/nrf91_dk_updating_fw_programmer.rst

```diff
@@ -15,7 +15,7 @@ The downloaded ZIP archive contains the following firmware:
 
 Application firmware
   The :file:`img_app_bl` folder contains full firmware images for different applications.
-  The guides in this section use the image for the :ref:`nrf_cloud_multi_service` sample as an example.
+  The guides in this section use the image for the nRF Cloud multi-service sample as an example.
```

Comment:
You are removing the sample, so why keep references to it in the user guides?

Reply:
Could replace with the AT client sample in this case (This was confirmed with PMT). FYI, This page is moved to the Tools page. I will be making the relevant changes on that page. We have PR open - https://github.com/nrfconnect/sdk-nrf/pull/25679, to remove this whole section and refer to the Tools page. For time being you could  replace this sample with the AT client sample.